info/v1/container.go

```diff
@@ -921,6 +921,24 @@ type ProcessStats struct {
 	// Ulimits for the top-level container process
 	Ulimits []UlimitSpec `json:"ulimits,omitempty"`
 }
+type PerfError struct {
```

Comment:
I think that all elements in PerfError struct could have a comment to keep style of  Perf struct.